cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseStreamAdmin.java

```diff
@@ -79,6 +78,11 @@ public boolean doTruncateTable(QueueName queueName) {
   }
 
   @Override
+  public void dropAllInNamespace(Id.Namespace namespace) throws Exception {
```

Comment:
QueueAdmin doesn't support methods on Id.Namespace; only on String. We discussed that that change would be a separate PR, since it turned out to be larger.